### DISCUSSION TITLE
Welcome to PreciousServices Discussions!

### DISCUSSION THREAD

**Discussion by SyedImtiyaz-1** [SyedImtiyaz-1/GetTechProjects#28] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by moizextra** [comment]
Hi! I want to ask that the PR's on this repo will be considered for Hacktoberfest

**Reply by SyedImtiyaz-1** [reply]
Yes, this repository comes under hacktober. But PR will only count from 1 Oct.
